libvcx/src/utils/constants.rs

```diff
@@ -114,6 +118,7 @@ pub static CRED_DEF_ID: &str = r#"2hoqvcwupRTUNkXn6ArYzs:3:CL:2471"#;
MjE4LDExNSwxOTUsMTY5LDE4OSw2Niw3MSwyMDcsNTAsMjcsNDgsMTU2LDE3OCwxODUsMjI3LDE0MCwyNywxNzAsMTEzLDE0OSwxMTcsMTgxLDc2LDE1OCwxMDcsMjM5XSxbMiw4LDExNywxMDEsMTM3LDE3MSwxMjUsMTcsMjM0LDEwOCwxOTksMjI0LDEyNiw3MCwxNjMsMTM3LDIyMCw0MSw1MCwxNDEsMTE5LDEyOCwzOSw4OSwyMDQsMTY5LDU0LDE1NywxNzcsMTY2LDI1MSwxMjEsMTQsOTQsODksMTUsMTA2LDEzMCwyMTgsMjE5LDExMSwxOTEsMTk3LDc0LDEzNSwyMjcsMTgwLDMyLDk5LDEwNyw4NSwyMDYsNDksODAsNDAsMTE5LDEyNCwyNDQsNzYsMjAzLDE3MCwxNjMsMTUxLDIyMCwyNTIsMTQ4LDk0LDkwLDQxLDE2OCwxNTAsMjIwLDE0NiwyNDgsOTEsMTAyLDEzMywyMjcsMTg5LDIxOSwyMDQsMTcsMTMsMTk2LDc2LDEwNCwxNzUsNTAsNjYsMTQzLDE2NiwxMjMsMTg1LDE3MCwxNzgsNjQsMTU0LDE1MSwxNjEsMjI4LDE4MSwxMjUsMTYzLDIwMCwyOSwxODUsMTMwLDE1LDE3OCwyNCwyMjQsMjIyLDE4MCwwLDE2NSwyNTQsMTMyLDQzLDU2LDE5MCwxNTcsMTcsMTkxLDcsMTQwLDE5NywxODQsMjQxLDEwNiwxMzQsMTc0LDk2LDE3MiwxNjEsNTAsMjI4LDIzNSwxMTIsMjIyLDE1MiwyMDMsOTYsNDUsMTQ4LDE0MiwxOSw5OSwyNTIsMTIsMjE4LDM5LDgyLDE3MSwyMzksMTY0LDY5LDIyLDE0MCwxMzIsMjI3LDE2NCwyNSw2NywxNjQsMjExLDQsMTUwLDEyLDE3OSwxNjcsMTEyLDEwMywxNDQsMywyMTcsMTQyLDIwNSwzMSwyNDAsMTA3LDIyLDIxMiw5MiwyMjcsNSwxMDksMTQyLDE1LDExMywyNDAsMTQ3LDEwMCw2MSwyMDcsNzIsMTkwLDIxMywyMDIsMTYsMjAyLDc5LDEzNCwyMzYsMTY4LDM2LDM4LDM0LDE2Niw1MiwxMzksNDQsMTAwLDc1LDk4LDI0NSwxMjcsMTUyLDM0LDEwOSwxMzMsMTY1LDExOSwxMDgsMjAxLDgzLDIyMSw2OSwxOTMsMjM5LDIyOSwyMiwzNCw0NiwzMyw0MSwyNDAsMjYsMjE4LDEwNSw1NywxMzUsMjQ2LDIzNCwyNDEsMTM5LDE3OCwyMjQsMTQ4LDUsMTk0LDEzMywyMzcsMTMsMTE2LDcsMjExLDc1XV19fSwicmVxdWVzdGVkX3Byb29mIjp7InJldmVhbGVkX2F0dHJzIjp7ImF0dHJpYnV0ZV8xIjp7InN1Yl9wcm9vZl9pbmRleCI6MCwicmF3IjoiMDUtMjAxOCIsImVuY29kZWQiOiIxMDEwODU4MTc5NTYzNzE2NDMzMTA0NzE4MjI1MzA3MTI4NDA4MzY0NDY1NzAyOTgxOTIyNzkzMDI3NTAyMzQ1NTQ4NDMzMzkzMjI4ODYifSwiYXR0cmlidXRlXzIiOnsic3ViX3Byb29mX2luZGV4IjowLCJyYXciOiJtYXRocyIsImVuY29kZWQiOiI3ODEzNzIwNDg3MzQ0ODc3Njg2MjcwNTI0MDI1ODcyMzE0MTk0MDc1NzAwNjcxMDgzOTczMzU4NTYzNDE0MzIxNTgwMzg0NzQxMDAxOCJ9fSwicmV2ZWFsZWRfYXR0cl9ncm91cHMiOnsiYXR0cmlidXRlXzAiOnsic3ViX3Byb29mX2luZGV4IjowLCJ2YWx1ZXMiOnsibGFzdF9uYW1lIjp7InJhdyI6ImNsYXJrIiwiZW5jb2RlZCI6IjUxMTkyNTE2NzI5Mjg3NTYyNDIwMzY4MjQyOTQwNTU1MTY1NTI4Mzk2NzA2MTg3MzQ1Mzg3NTE1MDMzMTIxMTY0NzIwOTEyMDgxMDI4In0sInNleCI6eyJyYXciOiJmZW1hbGUiLCJlbmNvZGVkIjoiNzE5NTcxNzQxNTYxMDgwMjI4NTc5ODU1NDM4MDY4MTY4MjAxOTg2ODAyMzMzODYwNDg4NDMxNzY1NjA0NzMyNDUxNTYyNDkxMTk3NTIifSwibmFtZSI6eyJyYXciOiJhbGljZSIsImVuY29kZWQiOiIxOTgzMTEzODI5Nzg4MDM2Nzk2Mjg5NTAwNTQ5NjU2MzU2MjU5MDI4NDY1NDcwNDA0NzY1MTMwNTk0ODc1MTI4NzM3MDIyNDg1NjcyMCJ9fX19LCJzZWxmX2F0dGVzdGVkX2F0dHJzIjp7ImF0dHJpYnV0ZV8zIjoiU21pdGgifSwidW5yZXZlYWxlZF9hdHRycyI6e30sInByZWRpY2F0ZXMiOnsicHJlZGljYXRlXzAiOnsic3ViX3Byb29mX2luZGV4IjowfX19LCJpZGVudGlmaWVycyI6W3sic2NoZW1hX2lkIjoiVjRTR1JVODZaNThkNlRWN1BCVWU2ZjoyOkZhYmVyVmN4OjYzLjgwLjgwIiwiY3JlZF9kZWZfaWQiOiJWNFNHUlU4Nlo1OGQ2VFY3UEJVZTZmOjM6Q0w6MTk6dGFnMSIsInJldl9yZWdfaWQiOiJWNFNHUlU4Nlo1OGQ2VFY3UEJVZTZmOjQ6VjRTR1JVODZaNThkNlRWN1BCVWU2ZjozOkNMOjE5OnRhZzE6Q0xfQUNDVU06dGFnMSIsInRpbWVzdGFtcCI6MTU5OTMzMjMzM31dfQ=="},"mime-type":"application/json"}],"~please_ack":{},"~thread":{"received_orders":{},"sender_order":0,"thid":"8399c71d-50a0-4844-8ac2-58ef08918bf7"}}"#;
```

Comment:
missing `ARIES` prefix, however besides there `ARIES_PROOF_PRESENTATION` in `mockdata_proof.rs`. If you depend on the data encoded within this mock, how about you add something like `ARIES_PROOF_PRESENTATION_SAMPLE2` to `mockdata_proof.rs`?